libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-overview.component.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component } from '@angular/core';
+import {
+  dhMeteringPointIdParam,
+  DhMeteringPointOverviewGuard,
+  dhMeteringPointPath,
+} from '@energinet-datahub/dh/metering-point/routing';
+import { render, RenderResult, screen } from '@testing-library/angular';
+import { SpectacularAppComponent } from '@ngworker/spectacular';
+
+import { DhMeteringPointOverviewModule } from './dh-metering-point-overview.module';
+import { DhMeteringPointOverviewComponent } from './dh-metering-point-overview.component';
+
+describe(DhMeteringPointOverviewComponent.name, () => {
+  beforeEach(async () => {
+    view = await render(SpectacularAppComponent, {
+      imports: [DhMeteringPointOverviewModule],
+      routes: [
+        {
+          path: dhMeteringPointPath,
+          children: [
+            {
+              canActivate: [DhMeteringPointOverviewGuard],
+              component: DhMeteringPointOverviewComponent,
+              path: `:${dhMeteringPointIdParam}`,
+            },
+          ],
+        },
+      ],
+    });
+  });
+
+  let view: RenderResult<SpectacularAppComponent>;
+  const meteringPointId = '571313180400014077';
+
+  it('displays a link to the Metering point URL', async () => {
+    await view.navigate(`/${dhMeteringPointPath}/${meteringPointId}`);
+
+    const link: HTMLAnchorElement = await screen.findByRole('link');
```

Comment:
suggestion: consider whether we should add more semantic meaning to the HTML structure, for example: should the breadcrumb contain a `<nav>` element?

Reply:
suggestion: consider what we are going to do when more links are added to this component. How can we identify this link amongst multiple `<a>` elements? Hint: include the metering point GSRN in the link query.

Reply:
> Hint: include the metering point GSRN in the link query.

What did you mean by this? The very first link (and the only one for now) doesn't include the GSRN in it. It only points to the metering point page. The GSRN is a plain text for now.

Reply:
Just pushed a commit that can support multiple links but only tests the first one for now.

Reply:
> What did you mean by this? The very first link (and the only one for now) doesn't include the GSRN in it. It only points to the metering point page. The GSRN is plain text for now.

Never mind. I remembered the navigation structure incorrectly but see comment in a different thread regarding this.